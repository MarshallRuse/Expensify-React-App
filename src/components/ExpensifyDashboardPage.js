import React from 'react';
import { Link } from 'react-router-dom';

import ExpenseList from './ExpenseList';
import ExpensesSummary from './ExpensesSummary';

const ExpensifyDashboardPage = () => (
    <div>
        <div className='page-header'>
            <div className='content-container'>
                <ExpensesSummary />
                <div className='page-header__actions'>
                    <Link className='button' to='/create'>Add Expense</Link>
                </div>
            </div>
        </div>
        <ExpenseList />
    </div>
);

export default ExpensifyDashboardPage;