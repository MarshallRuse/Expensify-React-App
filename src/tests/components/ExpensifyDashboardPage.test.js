import React from 'react';
import { shallow } from 'enzyme';
import ExpensifyDashboardPage from '../../components/ExpensifyDashboardPage';

test('Should render the ExpensifyDashboardPage component', () => {
    const wrapper = shallow(<ExpensifyDashboardPage />);
    expect(wrapper).toMatchSnapshot();
})