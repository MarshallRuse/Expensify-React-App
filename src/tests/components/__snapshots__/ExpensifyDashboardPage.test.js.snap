// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the ExpensifyDashboardPage component 1`] = `
<div>
  <div
    className="page-header"
  >
    <div
      className="content-container"
    >
      <[object Object] />
      <div
        className="page-header__actions"
      >
        <Link
          className="button"
          to="/create"
        >
          Add Expense
        </Link>
      </div>
    </div>
  </div>
  <[object Object] />
</div>
`;
